Content.Server/Chat/V2/Systems/RadioChannelValidationSystem.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Content.Shared.Chat.V2;
+using Content.Shared.Chat.V2.Components;
+using Content.Shared.Chat.V2.Systems;
+using Content.Shared.Radio;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Chat.V2.Systems;
+
+public sealed class RadioChannelValidationSystem : EntitySystem
```

Comment:
Another example of validation handling. Similar handlers will exist for sanitization, translation, yadda yadda. Note everything has access to ChatContext, so extending functionality can lean on the blob rather than changing event structure.